packages/backend/src/payment-method/ilp/connector/core/middleware/ilp-packet.ts

```diff
@@ -234,5 +235,34 @@ export function createIlpPacketMiddleware(
     ctx.assert(!ctx.body, 500, 'response body already set')
     ctx.assert(response.rawReply, 500, 'ilp reply not set')
     ctx.body = response.rawReply
+
+    if (ctx.services.telemetry && Number(prepare.amount)) {
+      ctx.services.telemetry?.incrementCounter('packet_count_prepare', 1, {
```

Comment:
optional chaining not necessary since we checked `ctx.services.telemetry`
```suggestion
      ctx.services.telemetry.incrementCounter('packet_count_prepare', 1, {
```